src/OpenTelemetry/CHANGELOG.md

```diff
@@ -32,6 +32,9 @@ Released 2022-Mar-30
   OpenTelemetryLoggerOptions after the LoggerFactory is built.
   ([#3055](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3055))
 
+* Add Copy method to both `MetricPoint` and `HistogramBuckets`.
```

Comment:
Do we need a changelog entry?

Reply:
I decided to err on the side of caution here.
I saw that other changelog entries included internal changes but weren't exclusively internal.

But no, this is an Internal only change so not needed.
Removed.